onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -289,7 +289,9 @@ class PlannerImpl {
 #endif
 
   // Find if there exists some input tensor that we can use in-place for output_arg_num-th input in the node.
-  bool FindReusableInput(const onnxruntime::Node& node, int output_arg_num, OrtValueIndex* reusable_input) {
+  bool FindReusableInput(const onnxruntime::Node& node, int output_arg_num, OrtValueIndex* reusable_input,
+                         bool* is_strided_tensor) {
+    *is_strided_tensor = false;
```

Comment:
Is maystrideinput/output compatible to use along with alias or others? If not, any guideline we need follow when we want to stride the input or output?

Reply:
Logically Alias, Inplace, StridedOutput have different meanings, an Op should not have 2 or more of them. Inplace means sharing the same Tensor, Alias means sharing same data buffer, and output tensor and shared tensor have same shape and strides. StridedOutput means sharing same data buffer, but output tensor has different shape or strides. You can think of Alias is a special case of strided tensor. Normally when you want an Op to support strided tensor as input/output, then use the StridedInput/StridedOutput, this can also cover the Alias case, so you don't need that Alias attribute anymore. And once it's strided tensor output, it's not possible to be Inplace.